pyLDAvis/gensim.py

```diff
@@ -61,14 +61,14 @@ def _extract_data(topic_model, corpus, dictionary, doc_topic_dists=None):
    return {'topic_term_dists': topic_term_dists, 'doc_topic_dists': doc_topic_dists,
            'doc_lengths': doc_lengths, 'vocab': vocab, 'term_frequency': term_freqs}
 
-def prepare(topic_model, corpus, dictionary, doc_topic_dist=None, **kwargs):
+def prepare(topic_models, corpus, dictionary, doc_topic_dist=None, **kwargs):
     """Transforms the Gensim TopicModel and related corpus and dictionary into
     the data structures needed for the visualization.
 
     Parameters
     ----------
-    topic_model : gensim.models.ldamodel.LdaModel
-        An already trained Gensim LdaModel. The other gensim model types are
+    topic_models : gensim.models.ldamodel.LdaModel
```

Comment:
If you are changing the core api to accept more than one model then you should allow the API to be backwards compatible, in the sense that the old functionality should remain even if just one model is provided.